Library/Formula/ceph.rb

```diff
@@ -0,0 +1,60 @@
+class Ceph < Formula
+  desc "Ceph is a scaleable distributed file system. Client tools/libraries only."
+  homepage "http://ceph.com/"
+  head "https://github.com/ceph/ceph.git"
+
+  depends_on "libtool" => :build
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl"
+  depends_on "snappy"
+  depends_on "cryptopp"
+  depends_on "boost"
+  depends_on "leveldb"
+
+  # Unfortunately it installs a Python script unconditionally, we depend on the
+  # Homebrew python so that when that script gets installed it gets installed
+  # into the Homebrew python site-packages rather than failing to install it in
+  # the system python site-packages.
```

Comment:
I am not familiar enough with autotools or this massive code base to even attempt to patch this, let alone understand what I need to modify to fix this. Hence the work-around.